diff --git a/node_modules/mermaid/dist/mermaid.core.mjs b/node_modules/mermaid/dist/mermaid.core.mjs
index 2ba23ae..d88a23c 100644
--- a/node_modules/mermaid/dist/mermaid.core.mjs
+++ b/node_modules/mermaid/dist/mermaid.core.mjs
@@ -69,7 +69,6 @@ var detector = /* @__PURE__ */ __name((txt) => {
   return /^\s*C4Context|C4Container|C4Component|C4Dynamic|C4Deployment/.test(txt);
 }, "detector");
 var loader = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/c4Diagram-5PPSVZJV.mjs");
   return { id, diagram: diagram2 };
 }, "loader");
 var plugin = {
@@ -129,7 +128,6 @@ var detector4 = /* @__PURE__ */ __name((txt) => {
   return /^\s*swimlane-beta\b/.test(txt);
 }, "detector");
 var loader4 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/swimlanesDiagram-ULZ7WXOC.mjs");
   return { id: id4, diagram: diagram2 };
 }, "loader");
 var plugin4 = {
@@ -161,7 +159,6 @@ var detector6 = /* @__PURE__ */ __name((txt) => {
   return /^\s*gitGraph/.test(txt);
 }, "detector");
 var loader6 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/gitGraphDiagram-DS77QQ5N.mjs");
   return { id: id6, diagram: diagram2 };
 }, "loader");
 var plugin6 = {
@@ -193,7 +190,6 @@ var detector8 = /* @__PURE__ */ __name((txt) => {
   return /^\s*info/.test(txt);
 }, "detector");
 var loader8 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/infoDiagram-6WML65LV.mjs");
   return { id: id8, diagram: diagram2 };
 }, "loader");
 var info = {
@@ -208,7 +204,6 @@ var detector9 = /* @__PURE__ */ __name((txt) => {
   return /^\s*pie/.test(txt);
 }, "detector");
 var loader9 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/pieDiagram-7S7Q4E2Y.mjs");
   return { id: id9, diagram: diagram2 };
 }, "loader");
 var pie = {
@@ -223,7 +218,6 @@ var detector10 = /* @__PURE__ */ __name((txt) => {
   return /^\s*quadrantChart/.test(txt);
 }, "detector");
 var loader10 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/quadrantDiagram-CIZ2JOQS.mjs");
   return { id: id10, diagram: diagram2 };
 }, "loader");
 var plugin8 = {
@@ -239,7 +233,6 @@ var detector11 = /* @__PURE__ */ __name((txt) => {
   return /^\s*xychart(-beta)?/.test(txt);
 }, "detector");
 var loader11 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/xychartDiagram-ELKLHX3M.mjs");
   return { id: id11, diagram: diagram2 };
 }, "loader");
 var plugin9 = {
@@ -255,7 +248,6 @@ var detector12 = /* @__PURE__ */ __name((txt) => {
   return /^\s*requirement(Diagram)?/.test(txt);
 }, "detector");
 var loader12 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/requirementDiagram-LRYGKXZP.mjs");
   return { id: id12, diagram: diagram2 };
 }, "loader");
 var plugin10 = {
@@ -366,7 +358,6 @@ var detector18 = /* @__PURE__ */ __name((txt) => {
   return /^\s*journey/.test(txt);
 }, "detector");
 var loader18 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/journeyDiagram-NVQOT4AX.mjs");
   return { id: id18, diagram: diagram2 };
 }, "loader");
 var plugin16 = {
@@ -455,7 +446,6 @@ var detector20 = /* @__PURE__ */ __name((txt) => {
   return /^\s*timeline/.test(txt);
 }, "detector");
 var loader20 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/timeline-definition-Z64GVDOM.mjs");
   return { id: id20, diagram: diagram2 };
 }, "loader");
 var plugin18 = {
@@ -471,7 +461,6 @@ var detector21 = /* @__PURE__ */ __name((txt) => {
   return /^\s*mindmap/.test(txt);
 }, "detector");
 var loader21 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/mindmap-definition-FAOFIHXS.mjs");
   return { id: id21, diagram: diagram2 };
 }, "loader");
 var plugin19 = {
@@ -487,7 +476,6 @@ var detector22 = /* @__PURE__ */ __name((txt) => {
   return /^\s*kanban/.test(txt);
 }, "detector");
 var loader22 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/kanban-definition-27J2QSJJ.mjs");
   return { id: id22, diagram: diagram2 };
 }, "loader");
 var plugin20 = {
@@ -503,7 +491,6 @@ var detector23 = /* @__PURE__ */ __name((txt) => {
   return /^\s*sankey(-beta)?/.test(txt);
 }, "detector");
 var loader23 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/sankeyDiagram-W5VNT64P.mjs");
   return { id: id23, diagram: diagram2 };
 }, "loader");
 var plugin21 = {
@@ -519,7 +506,6 @@ var detector24 = /* @__PURE__ */ __name((txt) => {
   return /^\s*packet(-beta)?/.test(txt);
 }, "detector");
 var loader24 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/diagram-LBJQPF4R.mjs");
   return { id: id24, diagram: diagram2 };
 }, "loader");
 var packet = {
@@ -534,7 +520,6 @@ var detector25 = /* @__PURE__ */ __name((txt) => {
   return /^\s*radar-beta/.test(txt);
 }, "detector");
 var loader25 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/diagram-UB23O5K3.mjs");
   return { id: id25, diagram: diagram2 };
 }, "loader");
 var radar = {
@@ -549,7 +534,6 @@ var detector26 = /* @__PURE__ */ __name((txt) => {
   return /^\s*block(-beta)?/.test(txt);
 }, "detector");
 var loader26 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/blockDiagram-VBNYF7ZC.mjs");
   return { id: id26, diagram: diagram2 };
 }, "loader");
 var plugin22 = {
@@ -565,7 +549,6 @@ var detector27 = /* @__PURE__ */ __name((txt) => {
   return /^\s*treeView-beta/.test(txt);
 }, "detector");
 var loader27 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/diagram-7IWD3JNH.mjs");
   return { id: id27, diagram: diagram2 };
 }, "loader");
 var plugin23 = {
@@ -581,7 +564,6 @@ var detector28 = /* @__PURE__ */ __name((txt) => {
   return /^\s*architecture/.test(txt);
 }, "detector");
 var loader28 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/architectureDiagram-T3A2C74G.mjs");
   return { id: id28, diagram: diagram2 };
 }, "loader");
 var architecture = {
@@ -597,7 +579,6 @@ var detector29 = /* @__PURE__ */ __name((txt) => {
   return /^\s*eventmodeling/.test(txt);
 }, "detector");
 var loader29 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/diagram-B4RE2ZJO.mjs");
   return { id: id29, diagram: diagram2 };
 }, "loader");
 var plugin24 = {
@@ -613,7 +594,6 @@ var detector30 = /* @__PURE__ */ __name((txt) => {
   return /^\s*ishikawa(-beta)?\b/i.test(txt);
 }, "detector");
 var loader30 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/ishikawaDiagram-WSZJBQD7.mjs");
   return { id: id30, diagram: diagram2 };
 }, "loader");
 var ishikawa = {
@@ -628,7 +608,6 @@ var detector31 = /* @__PURE__ */ __name((txt) => {
   return /^\s*venn-beta/.test(txt);
 }, "detector");
 var loader31 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/vennDiagram-T6HMQDX7.mjs");
   return { id: id31, diagram: diagram2 };
 }, "loader");
 var plugin25 = {
@@ -644,7 +623,6 @@ var detector32 = /* @__PURE__ */ __name((txt) => {
   return /^\s*treemap/.test(txt);
 }, "detector");
 var loader32 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/diagram-Q27KOJAE.mjs");
   return { id: id32, diagram: diagram2 };
 }, "loader");
 var treemap = {
@@ -659,7 +637,6 @@ var detector33 = /* @__PURE__ */ __name((text) => {
   return /^\s*wardley-beta/i.test(text);
 }, "detector");
 var loader33 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/wardleyDiagram-T6FBY63Y.mjs");
   return { id: id33, diagram: diagram2 };
 }, "loader");
 var plugin26 = {
@@ -675,7 +652,6 @@ var detector34 = /* @__PURE__ */ __name((txt) => {
   return /^\s*cynefin-beta(?:[\s:]|$)/.test(txt);
 }, "detector");
 var loader34 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/cynefinDiagram-MW4NZA55.mjs");
   return { id: id34, diagram: diagram2 };
 }, "loader");
 var cynefin = {
@@ -690,7 +666,6 @@ var detector35 = /* @__PURE__ */ __name((txt) => {
   return /^\s*railroad-beta/i.test(txt);
 }, "detector");
 var loader35 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/railroadDiagram-AXF67PYL.mjs");
   return { id: id35, diagram: diagram2 };
 }, "loader");
 var railroad = {
@@ -705,7 +680,6 @@ var detector36 = /* @__PURE__ */ __name((txt) => {
   return /^\s*railroad-ebnf-beta/i.test(txt);
 }, "detector");
 var loader36 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/ebnfDiagram-BXEA7PRR.mjs");
   return { id: id36, diagram: diagram2 };
 }, "loader");
 var railroadEbnf = {
@@ -720,7 +694,6 @@ var detector37 = /* @__PURE__ */ __name((txt) => {
   return /^\s*railroad-abnf-beta/i.test(txt);
 }, "detector");
 var loader37 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/abnfDiagram-N423BO3Z.mjs");
   return { id: id37, diagram: diagram2 };
 }, "loader");
 var railroadAbnf = {
@@ -735,7 +708,6 @@ var detector38 = /* @__PURE__ */ __name((txt) => {
   return /^\s*railroad-peg-beta/i.test(txt);
 }, "detector");
 var loader38 = /* @__PURE__ */ __name(async () => {
-  const { diagram: diagram2 } = await import("./chunks/mermaid.core/pegDiagram-VL7TDLO6.mjs");
   return { id: id38, diagram: diagram2 };
 }, "loader");
 var railroadPeg = {
